### PR TITLE
Pixel monkeys and hostile simple animals

### PR DESCRIPTION
**Pull request by Couls**
They'll now properly path at you at a reasonable pace

### CODE REVIEW

File: code/modules/mob/mob_movespeed.dm [hunk 1]
```diff
@@ -108,7 +108,7 @@ Key procs
 				conflict_tracker[conflict] = amt
 			else
 				continue
-		. *= amt
+		. += amt
```
Comment (Couls) [inline code/modules/mob/mob_movespeed.dm:111]: dunno if you want this changed just yet, lemme know

File: code/modules/mob/living/simple_animal/hostile/hostile.dm [hunk 2]
```diff
@@ -56,6 +56,9 @@
 
 	if(!targets_from)
 		targets_from = src
+	//PIXEL MOVEMENT
+	move_to_delay = move_to_delay * 0.1
```
Comment (Couls) [inline code/modules/mob/living/simple_animal/hostile/hostile.dm]: probably want to eventually remove move_to_delay entirely

File: code/modules/mob/living/simple_animal/hostile/hostile.dm [hunk 3]
```diff
@@ -56,6 +56,11 @@
 
 	if(!targets_from)
 		targets_from = src
+	//PIXEL MOVEMENT
+	move_to_delay *= 0.1
+	aggro_vision_range *= 32 // 32 pixels per turf
+	vision_range *= 32
```
Comment (ninjanomnom) [inline code/modules/mob/living/simple_animal/hostile/hostile.dm]: I would rather we modify all these ourselves in this branch

**Review by ninjanomnom — CHANGES_REQUESTED** [review]

File: code/modules/mob/mob_movespeed.dm (hunk 1 above)
Comment (ninjanomnom) [reply code/modules/mob/mob_movespeed.dm:111]: Yeah this is fine, I'll be messing with movespeed in a bit

**Pull request merged**